Build/Scripts/CompileAppInBcContainer.ps1

```diff
@@ -36,7 +36,7 @@ $CICDBuild = $env:GITHUB_WORKFLOW -and ($($env:GITHUB_WORKFLOW).Trim() -eq 'CI/C
 
 if($CICDBuild) {
     # Create the artifacts folder for the app to place in the package
-    . $PSScriptRoot\CreateAppPackageOutput.ps1 -AppProjectFolder $parameters["appProjectFolder"] -BuildMode $appBuildMode -AppFile $appFile -ALGoProjectFolder $currentProjectFolder -IsTestApp:$(!$app)
+    . $PSScriptRoot\Package\CreateAppPackageOutput.ps1 -AppProjectFolder $parameters["appProjectFolder"] -BuildMode $appBuildMode -AppFile $appFile -ALGoProjectFolder $currentProjectFolder -IsTestApp:$(!$app)
```

Comment:
Shouldn't that be called based on the info in `Package.json`?

As in... if the value for `IncludeInPackage` is _true_ for the current app, only then run `CreateAppPackageOutput.ps1`.